pennylane/measurements/shots.py

```diff
@@ -161,6 +161,20 @@ def __deepcopy__(self, memo):
         memo[id(self)] = self
         return self
 
+    def __str__(self):
+        """The string representation of the class"""
+
+        shot_copy_strs = [
+            f"({sc.shots} shots x {sc.copies})" if sc.copies > 1 else f"{sc.shots}"
+            for sc in self.shot_vector
+        ]
+        shot_copy_str = ", ".join(shot_copy_strs) or None
+        return f"Shots(total={self.total_shots}, shot_vect=[{shot_copy_str}])"
```

Comment:
```suggestion
        return f"Shots(total={self.total_shots}, shot_vector=[{shot_copy_str}])"
```
Typo or deliberate?

Reply:
On purpose 👍🏼

Reply:
If we want bevity, maybe `vector`?

Reply:
I would prefer the full name, we can afford the 2 letters

Reply:
Added 👍🏼